services/src/main/java/io/grpc/services/HealthCheckingLoadBalancerUtil.java

```diff
@@ -63,7 +64,9 @@ private HealthCheckingLoadBalancerUtil() {
   public static LoadBalancer newHealthCheckingLoadBalancer(Factory factory, Helper helper) {
     HealthCheckingLoadBalancerFactory hcFactory =
         new HealthCheckingLoadBalancerFactory(
-            factory, new ExponentialBackoffPolicy.Provider(), TimeProvider.SYSTEM_TIME_PROVIDER);
+            factory, new ExponentialBackoffPolicy.Provider(), TimeProvider.SYSTEM_TIME_PROVIDER,
+            Stopwatch
+                .createUnstarted());
```

Comment:
nit: `TimeProvider.SYSTEM_TIME_PROVIDER,` in a new line, `Stopwatch.createUnstarted()` in a line.